### PR TITLE
Add URL Column

### PR DESCRIPTION
**Pull request by EGYWEB-Mohamed**
Add URL Column Details

### CODE REVIEW

File: 5.x/crud-columns.md [hunk 1]
```diff
@@ -599,6 +599,20 @@ The text column will just output the text value of a db column (or model attribu
     'type'  => 'text'
 ],
 ```
+<hr>
+
+<a name="url"></a>
+### url
+
+The text column will just output the url value of a db column. Its definition is:
```
Comment (pxpm) [inline 5.x/crud-columns.md:607]:
```suggestion
The `url` column will just output the url value of a db column. Its definition is:
```

**Review by pxpm — CHANGES_REQUESTED** [review]
Thanks for the PR @EGYWEB-Mohamed 

Can you please just move the `url` column docs you created after the `textarea` so it stays "first letter alphabetically ordered" ? 

Thanks

**Comment by EGYWEB-Mohamed** [comment]
@pxpm Done, please notice this doc in favour of this pull request : https://github.com/Laravel-Backpack/CRUD/pull/4925

**Comment by karandatwani92** [comment]
This will be available for backpack v6

**Pull request closed without merging**